init_db.py

```
@@ -6,13 +6,15 @@
 
 import os
 import sys
+
 from sqlalchemy import inspect
 
 # Add the app directory to the path
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'app'))
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "app"))
 
 from app import app, db  # pylint: disable=wrong-import-position
```

Pylint (4.0.4)

[error] No name 'db' in module 'app'

(E0611)